coralproject/talk
# [CORL-3106]: Update notifications feed styles

## What does this PR do?

These changes update the notifications feed to match updated designs. It streamlines the notification bodies, makes the full notification body clickable to a permalink view of the comment, adds badges for staff members replying, updates embeds to only show a tag when present, truncates comment text after 120 characters, and updates icons.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags?

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

You can test this PR to matching it against the updated designs and seeing that it looks as expected. Also click on each notification in the feed and see that you have a new tab opened to the relevant comment.

## Were any tests migrated to React Testing Library?



## How do we deploy this PR?
